ouster_client/src/client.cpp

```diff
@@ -328,9 +328,19 @@ std::shared_ptr<client> init_client(const std::string& hostname,
     std::string res;
     bool success = true;
 
+  // If udp_dest_host is empty string, use automatic addressing with set_udp_dest_auto
+  if (udp_dest_host != "")
```

Comment:
minor nitpick: swapping the if/else branches and checking`udp_dest_host == ""` would be preferable

Reply:
Next time. Thanks for the merge.